content/08.outlook.md

```diff
@@ -1 +1,45 @@
+
+Mindful addition of data is key for powering the next generation of analysis in rare disease data.
+While there are many techniques to collate rare data from different sources, incorrect data generation may hurt the end goal even if it adds to the size of the dataset.
+In our experience collaboration with domain experts have proved to be critical in gaining insight into potential sources of variation in the datasets.
+As an example, an neurofibromatosis type 1 (NF1) dataset was found to contain samples collected using vastly different surgical techniques (laser ablation and excision vs standard excision). [@doi:10.3390/genes11020226] 
+While the integrative analysis in the study using transfer learning techniques was able to minimize technique related signals [@doi:10.1016/j.cels.2019.04.003], a more traditional analysis may have resulted in surfacing of substantial biological differences that are a consequence of process (e.g. activation of heat shock protein related pathways), not disease related biology. 
+Such instances underline the fact that continuous collaboration with domain experts is needed to generate robust datasets in the future.
+A few such collaborations are beginning to show promise in generating valuable datasets for future use.[@https://www.hopkinsmedicine.org/kimmel_cancer_center/cancers_we_treat/pediatric_oncology/research_and_clinical_trials/pratilas/nf1_biospecimen_repository.html]
+
+In addition to sample scarcity, there is a dearth of comprehensive phenotypic-genotypic databases in rare disease.
+With the ubiquity of sequencing platforms, genomic data has been easy to gather for rare disease patients.[@doi:10.1038/nrg3555; @doi:10.1038/nrg.2017.116; @doi:10.1056/NEJMra1711801]
+An important next step would be to integrate genomic and clinical data to produce comprehensive phenotypic-genotypic databases that can fuel interpretation of features extracted using ML methods.
+Finally, mindful sharing of data with proper metadata and attribution to enable prompt data reuse is of utmost important in building datasets that can be of great value in rare disease. [@https://www.nature.com/articles/s41576-020-0257-5]
+
+
+_Development of methods that reliably support mechanistic interrogation of specific rare diseases_
+
+A pattern that emerged readily in our quest for examples of application of ML methods in rare disease is that most ML related strategies were focused on classification of a patient as a rare disease patient among other non-rare disease patients.
+We found that examples of methodologies interrogating mechanistic details of a specific rare disease were particularly lacking.
+Reasons for this include not only the lack of large datasets, but also lack of reliability or interpretability of the results from such analyses.
+We believe that an intentional push towards developing methods or analytical workflows that address this would be critical in encouraging powerful analyses using rare disease data.
```

Comment:
```suggestion
An intentional push towards developing methods or analytical workflows that address this will be critical to apply machine learning approaches to rare disease data.
```